docs/admin/logical-replication.rst

```diff
@@ -128,3 +128,13 @@ superuser is allowed to ALTER or DROP a subscription.
    A network setup that allows the two clusters to communicate is a
    pre-requisite for a working publication/subscription setup.
    See :ref:`HBA <admin_hba_node>`.
+
+Monitoring
+----------
+
+Existing publications can be checked by querying :ref:`pg_publication` table
+which contains general information about a publication.
+:ref:`pg_publication_tables` contains list of replicated tables for a specific
+publication. :ref:`pg_subscription` table contains general information about a
+subscription. :ref:`pg_subscription_rel` table has replication state detail per
+table including error message if there was an error.
```

Comment:
```suggestion
All publications are listed in the :ref:`pg_publication` table.
More details for a publication are available in the
:ref:`pg_publication_tables` table. It lists the replicated tables for a
specific publication.

All subscriptions are listed in the :ref:`pg_subscription` table.
More details for a subscription are available in the:ref:`pg_subscription_rel`
table. The table contains detailed information about the replication state,
including error messages if there was an error.
```

Just a suggestion to reduce the passive voice a bit. Not sure if it is better - up to you.

Reply:
I'm applying the suggestion - one minor note about last sentence - shouldn't I keep replication state **per
table** to emphasise the granularity?

Reply:
> shouldn't I keep "per table" to emphasize the granularity?

yes, good point :+1: